modules/cli/src/main/scala/scala/cli/launcher/PowerOptions.scala

```diff
@@ -0,0 +1,21 @@
+package scala.cli.launcher
+
+import caseapp.*
+
+import scala.cli.commands.shared.HelpGroup
+import scala.cli.commands.tags
+
+// format: off
```

Comment:
Actually format off helps in our option classes since fmt does not indent the option name further thatn annotations and I think it's hard to read with many options:
preferred:
```
case class PowerOptions(
   @Group(HelpGroup.Launcher.toString)
   @HelpMessage("When called as 'scala', allow to use power commands too")
   @Tag(tags.must)
     power: Boolean = false
 )
```

fmt format:
```
case class PowerOptions(
   @Group(HelpGroup.Launcher.toString)
   @HelpMessage("When called as 'scala', allow to use power commands too")
   @Tag(tags.must)
   power: Boolean = false
 )
```